resources/home/dnanexus/annotate_workbooks/annotate_workbooks_with_QC.py

```
@@ -159,11 +184,117 @@ def create_combined_qc(multiqc_path):
+
+        for gene, depth in gene_depths.items():
+            pos = gene_pos[gene]
+            depth_result, pos_result = write_gene_depth_to_cell(
+                worksheet,
+                gene,
+                depth=depth,
+                pos=pos)
+            if depth_result is None:
+                print(f"[WARN] Skipped {gene}: no cell location")
+        sample_workbook.save(workbook_path)
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)